pkg/kubelet/dockertools/manager.go

```diff
@@ -1861,7 +1972,94 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, _ api.PodStatus, podStatus *kubec
 		}
 	}
 
-	// Start everything
+	next, status, done := findActiveInitContainer(pod, podStatus)
```

Comment:
each invocation of SyncPod will only start one container, and there's a lag between invocations? Or did I miss a for loop somewhere here. 

I think this runs in a podworker goroutine, if that's the case we should just be able to wait for all init's to start/exit synchrounsly?